staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt.go

```diff
@@ -240,6 +241,7 @@ func (reqMgmt *requestManagementSystem) Wait(requestDigest RequestDigest) (bool,
 			matchingpriorityLevelName = fs.Spec.PriorityLevelConfiguration.Name
 			matchingFlowDistinguisherMethod = fs.Spec.DistinguisherMethod
 		default: // reject
+			metrics.RecordRejects("", "non-match")
```

Comment:
Perhaps a non-empty priority level name here would be good?  We can certainly use a string that is not a valid kube object name.

Reply:
updated to `<none>`